### DISCUSSION TITLE
nginx unit and auto reload...?

### DISCUSSION THREAD

**Discussion by ziggekatten** [netbox-community/netbox-docker#473] [Getting Help]
The new way of using Unit have some benefits, especilly stability when doing lot of API calls (now at 100%)

However, I struggle to figure out how to make unit auto reload, as I develop alot of various reports and scripts, and this was a simple parameter in the gunicorn days.

How have you solved this?

**Comment by ziggekatten** [comment]
Bumping this as I have not found an solution other than completely revert from Unit to Gunicorn

**Comment by hl037** [comment]
bumping too...

**Comment by tobiasge** [comment]
You can restart an application inside of Unit. See their [documentation](https://unit.nginx.org/configuration/#process-management).

**Comment by timrabl** [comment]
```bash
user@srv01 ~ % docker-compose exec netbox curl -X GET --unix-socket /opt/unit/unit.sock http://localhost/control/applications/netbox/restart
{
	"success": "Ok"
}
